src/lib/util.js

```diff
@@ -1,8 +1,9 @@
 export const VOID_ELEMENTS = /^(?:area|base|br|col|embed|hr|img|input|link|meta|param|source|track|wbr)$/;
 export const UNSAFE_NAME = /[\s\n\\/='"\0<>]/;
 export const NAMESPACE_REPLACE_REGEX = /^(xlink|xmlns|xml)([A-Z])/;
-export const HTML_LOWER_CASE = /^accessK|^auto[A-Z]|^cell|^ch|^col|cont|cross|dateT|encT|form[A-Z]|frame|hrefL|inputM|maxL|minL|noV|playsI|popoverT|readO|rowS|spellC|src[A-Z]|tabI|useM|item[A-Z]/;
-export const SVG_CAMEL_CASE = /^ac|^ali|arabic|basel|cap|clipPath$|clipRule$|color|dominant|enable|fill|flood|font|glyph[^R]|horiz|image|letter|lighting|marker[^WUH]|overline|panose|pointe|paint|rendering|shape|stop|strikethrough|stroke|spel|text[^L]|transform|underline|unicode|units|^v[^i]|^w|^xH/;
+export const HTML_LOWER_CASE = /^accessK|^auto[A-Z]|^cell|^ch|^col|cont|cross|dateT|encT|form[A-Z]|frame|hrefL|inputM|maxL|minL|noV|playsI|popoverT|readO|rowS|src[A-Z]|tabI|useM|item[A-Z]/;
+export const HTML_ENUMERATED = /^dra|spel/;
```

Comment:
Can we use a set instead, performance wise this could help a lot

Reply:
Sure, can do.

Reply:
Out of curiosity, what makes a set preferable here and not for the others? We certainly do make use of the patterns in a couple places, but that should just be byte saving, no?

There are certainly are a few more boolean props -> enumerated attrs, if it's the size of 2 that is what immediately concerns you, though I don't know quite how many there are. Will have to go hunting.

Reply:
I have undergone the efforts to port self-closing to a set but for the others I just don't know the amount of cases 😅

Reply:
Gotcha, thanks!